DBMS/DBMSSession5/StoreFrontQueries.java

```diff
@@ -0,0 +1,76 @@
+package GET2018.DBMS.DBMSSession5;
+
+/**
+ * This class stores various Queries on database
+ * 
+ * @author Mohit Sharma
+ *
+ */
+public class StoreFrontQueries {
```

Comment:
Do we really need setter methods and variables in this class. I think getter method would be sufficient